### DISCUSSION TITLE
Vben下载下来如何能运行起来？

### DISCUSSION THREAD

**Discussion by zqhgit001** [vbenjs/vue-vben-admin#1185] [General]
求助，Vben 整了一天也运行不起来啊！！！

**Comment by M69W** [comment]
网络原因？
如果git clone 不行，就直接下载也行

**Reply by zqhgit001** [reply]
Vben 是不是前后端分离，怎么才能运行起来？

**Reply by M69W** [reply]
不是，用的是mock数据

**Reply by zqhgit001** [reply]
老铁，我搞不定，运行不起来，可以加你QQ吗？，我的QQ：68383913      谢谢！

**Reply by zqhgit001** [reply]
老铁，如何绑定后端？

**Reply by zqhgit001** [reply]
老铁，后台用的是啥？是不是webapi？

**Reply by dievdy** [reply]
装node和yarn，安装包啊，然后执行命令
https://vvbin.cn/doc-next/guide/introduction.html#%E9%9C%80%E8%A6%81%E6%8E%8C%E6%8F%A1%E7%9A%84%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86

**Comment by zqhgit001** [comment]
老铁，后台用的是啥？是不是webapi？
